src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx

```diff
@@ -28,26 +28,29 @@ To uninstall agents or integrations, here are some recommended procedures:
 * [Remove infrastructure agent](/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/uninstall-infrastructure-agent/)
 * For how to disable other New Relic tools, see their specific docs. You can [search New Relic solutions here](https://newrelic.com/instant-observability/).
 
-## Downgrade organization [#downgrading]
+## Downgrade your edition [#downgrading]
 
-Options for downgrading your New Relic organization will differ depending on your pricing edition:
+Note that your [New Relic edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) applies to your entire [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure). 
+
+Options for downgrading New Relic: 
 
 <CollapserGroup>
   <Collapser
-    id="downgrade-pro-enterprise"
-    title="Pro or Enterprise edition downgrade"
+    id="payg-downgrade"
+    title="Pay-as-you-go organizations"
   >
-    To downgrade your Pro or Enterprise organization, contact your New Relic account representative.
+To downgrade a Standard or Pro [pay-as-you-go](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans) organization: 
+
+1. From the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), click **Manage your plan**. 
+2. Click **Plans and pricing**. 
+3. Under the **Free** pricing option, hover above **Downgrade to free** and click the button that appears.
```

Comment:
Rather than `hover above` I believe this should be `mouse over` ( https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#mouse-over )

Reply:
+1

Reply:
Also, does the "button that appears" not have a name? If it does, I'd just do something like "...mouse over Downgrade to free, then click the such-and-such button"